tests/system/action/motion/test_set_support_self.py

```diff
@@ -0,0 +1,193 @@
+                "meeting_id": 1,
+            },
+        )
+        self.create_model(
+            "meeting/1",
+            {
+                "name": "name_meeting_1",
+                "motion_ids": [1],
+                "motions_supporters_min_amount": 0,
+            },
+        )
+        response = self.client.post(
+            "/",
+            json=[
+                {
+                    "action": "motion.set_support_self",
+                    "data": [{"motion_id": 1, "support": True}],
+                }
+            ],
+        )
+        self.assert_status_code(response, 400)
+        assert "Motion supporters system deactivated." in response.data.decode()
+
+    def test_state_doesnt_allow_support(self) -> None:
+        self.create_model(
+            "motion/1",
+            {
+                "title": "motion_1",
+                "meeting_id": 1,
+                "state_id": 1,
+            },
+        )
+        self.create_model(
+            "meeting/1",
+            {
+                "name": "name_meeting_1",
+                "motion_ids": [1],
+                "motions_supporters_min_amount": 1,
+            },
+        )
+        self.create_model(
+            "motion_state/1",
+            {
+                "name": "state_1",
+                "allow_support": False,
+                "motion_ids": [1],
+                "meeting_id": 1,
+            },
+        )
+        response = self.client.post(
+            "/",
+            json=[
+                {
+                    "action": "motion.set_support_self",
+                    "data": [{"motion_id": 1, "support": True}],
+                }
+            ],
+        )
+        self.assert_status_code(response, 400)
+        assert "The state does not allow support." in response.data.decode()
+
+    def test_support(self) -> None:
+        self.create_model(
+            "motion/1",
+            {
+                "title": "motion_1",
+                "meeting_id": 1,
+                "state_id": 1,
+                "supporter_ids": [],
+            },
+        )
+        self.create_model(
+            "meeting/1",
+            {
+                "name": "name_meeting_1",
+                "motion_ids": [1],
+                "motions_supporters_min_amount": 1,
+            },
+        )
+        self.create_model(
+            "motion_state/1",
+            {
+                "name": "state_1",
+                "allow_support": True,
+                "motion_ids": [1],
+                "meeting_id": 1,
+            },
+        )
+        response = self.client.post(
+            "/",
+            json=[
+                {
+                    "action": "motion.set_support_self",
+                    "data": [{"motion_id": 1, "support": True}],
+                }
+            ],
+        )
+        self.assert_status_code(response, 200)
+        model = self.get_model("motion/1")
+        assert model.get("supporter_ids") == [1]
```

Comment:
Please assert that the reverse relation is set correctly.

Reply:
Added a reverse relation check.

Reply:
Also check that the template field is set correctly (`supported_motion_$_ids == ["1"]`)